src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ CHANGELOG
 -----
 
  * Trigger `entered` event for subject entering in the Workflow for the first time
+ * Added dump styling of states and transitions through `metadata`,
+   configuration of places is:
+   `label` and `description` (only for PUML) (text), `bg_color` (`#RRGGBB` or CSS color name)
```

Comment:
hello, i think it will be better to prefix by **dump_** those metadata, as they are specific for the dump feature
wdyt?

Reply:
This is a good question because it may conflict with an existing configs in some projects. We chose to use a simple syntax: https://github.com/symfony/symfony/pull/29538#discussion_r241770962